sdk/python/kfp/dsl/type_utils.py

```diff
@@ -40,6 +40,10 @@
     'string': pipeline_spec_pb2.PrimitiveType.STRING,
     'str': pipeline_spec_pb2.PrimitiveType.STRING,
     'text': pipeline_spec_pb2.PrimitiveType.STRING,
+    'bool': pipeline_spec_pb2.PrimitiveType.STRING,
+    'boolean': pipeline_spec_pb2.PrimitiveType.STRING,
+    'dict': pipeline_spec_pb2.PrimitiveType.STRING,
```

Comment:
Should we also add `JSONArray` `JSONBlob` etc?

Reply:
Done.